File: activity/templates/app-java/src/main/java/di/modules/_Module.java
package <%= appPackage %>.di.modules;

import dagger.Module;
import dagger.Provides;
import <%= appPackage %>.ui.<%= activityPackageName %>.<%= activityName %>Activity;

// android-hipster-needle-module-provides-import

@Module
public class <%= activityName %>Module extends ActivityModule {

    public <%= activityName %>Module(<%= activityName %>Activity activity) {
        super(activity);
    }

    // android-hipster-needle-module-provides-method

}

File: activity/templates/app-kotlin/src/main/java/di/modules/_Module.kt
package <%= appPackage %>.di.modules

import dagger.Module
import dagger.Provides
import <%= appPackage %>.ui.<%= activityPackageName %>.<%= activityName %>Activity

// android-hipster-needle-module-provides-import


@Module
class <%= activityName %>Module(val activity: <%= activityName %>Activity) {

    @Provides
    fun provideActivity() : <%= activityName %>Activity{
        return activity
    }

    // android-hipster-needle-module-provides-method

}
